### PR TITLE
Install djangoapp for Stanford's data reports

### PR DESCRIPTION
**Pull request by stvstnfrd**

### CODE REVIEW

**Comment by stvstnfrd** [comment]
This is currently installed on stage (confirmed to work) and resolves the report generation issues.
I'd like someone else to spot check, but my reports go through now.

**Comment by stvstnfrd** [comment]
For transparency, I did test this locally before PR/merging.
I suspect I didn't catch the issue since devstack doesn't run tasks via celery.

**Comment by stvstnfrd** [comment]
@caesar2164 @kluo Please review the new additions. This is best reviewed by inspecting individual commits, one by one.

File: openedx/stanford/lms/djangoapps/instructor_task/tasks_helper.py [hunk 1]
```diff
@@ -218,8 +220,8 @@ def update_task_progress():
         header, datarows = csv_fn(course_id)
         rows = [header] + [row for row in datarows]
     # Update progress to failed regardless of error type
-    # pylint: disable=bare-except
```
Comment (caesar2164) [inline openedx/stanford/lms/djangoapps/instructor_task/tasks_helper.py]: why are removing this pylint disabling?

Comment (stvstnfrd) [reply openedx/stanford/lms/djangoapps/instructor_task/tasks_helper.py]: Take another look at _which_ pylint error is removed...

Comment (caesar2164) [reply openedx/stanford/lms/djangoapps/instructor_task/tasks_helper.py]: oh DOH, ok nvmd

**Pull request merged**